src/main/js/dynamic/ChannelTestModal.js

```diff
@@ -1,14 +1,24 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Modal } from 'react-bootstrap';
-import TextInput from 'field/input/TextInput';
+import FieldsPanel from 'field/FieldsPanel';
+import * as FieldModelUtilities from 'util/fieldModelUtilities';
+import * as FieldMapping from "../util/fieldMapping";
```

Comment:
Relative path should be absolute

Reply:
I thought I got all of them.  I missed this one. Thanks.